### PR TITLE
fix - cannot comment

### PR DESCRIPTION
**Pull request by zendobk**
#2 linterbot doesn't post any warnings to GitHub pull request

### CODE REVIEW

**Comment by zendobk** [comment]
#2

File: lib/linterbot/comment_generator.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def find_file
       end
 
       def new_file?
-        file.staus == "added"
+        file.status == "added"
```
Comment (at-binhcq) [inline lib/linterbot/comment_generator.rb:40]: Use `''` for `String` instead of `""`

**Comment by guidomb** [comment]
Thanks for the contribution and sorry for the delayed response!

**Comment by at-daonguyen** [comment]
I'm excited to see your response!

**Comment by guidomb** [comment]
I mean sorry for taking so long to merge this patch ;-)

**Pull request merged**